### ISSUE TITLE
Tabs considerations

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3669]
The following is an omnibus issue for the Tabs component. It is a summary of the main considerations surfaced in a review of the carbondesignsytem.com guidance. A more thorough set of comments can be viewed in the commented documents, [tabs usage.pdf](https://ibm.box.com/s/fb5jn3v0fpqeue4ta2ss5s4fitjvipul) and [tabs style.pdf](https://ibm.box.com/s/dl1a1b7pzef6nuct3hqlimc1rqd7cfrj).

The same two main considerations for Tabs are the same already flagged in an [issue for Content Switcher](https://github.com/carbon-design-system/carbon-website/issues/3364)
- Lack of styling guidance on tabpanel
- No mention of manual versus automatic tab selection

### Lack of styling guidance on tabpanel
On many of the examples in the usage page, there is a clear visual tab panel designation using the white area on a grey background. It ties in the selected tab item with its panel. This is a basic feature of tab panels (especially the 'contained' version) yet there is no documentation about it on either the style or usage pages.

### No description of manual versus automatic tab selection
I've added information on this into the [draft Accessibility tab info](https://carbondesignsystem-mezlgs4qw-carbon-design-system.vercel.app/components/tabs/accessibility). I think it is reasonable for this to be discussed on the Usage page as well.

**Comment by kingtraceyj** [comment]
**Usage updates:**
-Added content switcher information on when not to use a tab
-Added scroll button to anatomy section
-Added clarity around scrolling tabs and overflow content (more to come with vertical tab updates)
-Updated information about navigating away tab and returning
-Added information about automatic verses manual tabs (gif is coming! trying to troubleshoot it and it was a blocker)
-Updated icon placement to include dismissible tab variant parameters

**Style updates:**
-Took out scroll focus states and opened
-Discussed darkening unselected line in default tabs to meet 3:1 contrast but the contrast was too similar to selected states and decided to keep as is

**Continued exploration:**
-Investigate line extension to end of the tab panel for default tabs—this seems to be a big change for some product teams but many other systems include this in similar types of tabs
-[Investigate focus state](https://www.figma.com/design/qGVzyBMsr4nR7clX6rnZOe/Tabs?node-id=17-198013&t=KjllHL5pQ84IPKBq-1) going around whole tablist [#15962](https://github.com/carbon-design-system/carbon/issues/15962) (explorations started but not complete)
